### ISSUE TITLE
Re-integrate SPS selenium install in preparation of SPS being added into the marketplace

### ISSUE THREAD

**Issue by hargitayjpl** [unity-sds/unity-cs#361]

**Comment by hargitayjpl** [comment]
https://github.com/unity-sds/unity-cs-infra/pull/74

PR

**Comment by hargitayjpl** [comment]
**ISSUE**: All the buttons/html elements in the Management console marketplace have the same class name. Sometimes the selenium scripts clicks the wrong button due to this.

**Comment by hargitayjpl** [comment]
```
Test Name                     | Status
------------------------------+--------
test_navigate_to_mc_console   | ✔
test_bootstrap_process_status | ✔
test_initiate_core_setup      | ✔
test_core_setup_save_btn      | ✔
test_naviate_to_SPS_module    | ✖
test_SPS_module_name          | ✖
test_SPS_module_branch        | ✖
test_click_SPS_install_btns   | ✖
test_SPS_install_status       | ✖
```

The installation of SPS is failing as expected until SPS is integrated into the MC marketplace.

**Comment by GodwinShen** [comment]
@hargitayjpl and @galenatjpl  this needs an estimate.  Also, when do we expect this ticket to get un-blocked and which ticket unblocks it?

**Comment by hargitayjpl** [comment]
@GodwinShen - Once #351 is completed and SPS is added to the Management Console Marketplace, I'll be able to add to our selenium nightly script a function that tests the install and uninstall of SPS. I'll also add an estimated time to finish this task to the ticket.

**Comment by galenatjpl** [comment]
@GodwinShen @hargitayjpl @jpl-btlunsfo @mike-gangl I added a "dependency" tag on #351 , that references this ticket, and also updated that ticket's description.

@hargitayjpl is the work needed to solve "sometimes the selenium clicks the wrong button" done?  Was there a separate ticket for @jdrodjpl about this?

**Comment by hargitayjpl** [comment]
@galenatjpl and @jdrodjpl - There is no ticket for that issue however work does need to be done. Most of the Buttons in the MC share the same HTML Class/ID name. This makes it ambiguous for the Selenium script to select the correct button.

**Comment by galenatjpl** [comment]
@hargitayjpl I think that https://github.com/unity-sds/unity-cs/issues/363 solved this, correct?  It looks like the change by @jdrodjpl was merged in https://github.com/unity-sds/unity-management-console/pull/13 .

**Comment by hargitayjpl** [comment]
@galenatjpl Yea, that ticket solves it!!!
